examples/specs/bar_custom_sort.vl.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A simple bar chart with embedded data.",
+  "data": {
+    "values": [
+      {"a": "A","b": 28}, {"a": "B","b": 55}, {"a": "C","b": 43},
+      {"a": "D","b": 91}, {"a": "E","b": 81}, {"a": "F","b": 53},
+      {"a": "G","b": 19}, {"a": "H","b": 87}, {"a": "I","b": 52}
+    ]
+  },
+  "mark": "bar",
+  "encoding": {
+    "x": {"field": "a", "type": "ordinal", "sort": ["D", "G", "F"]},
```

Comment:
Maybe it's better distinguish between (1) the example that show how sort order normally works when all items are included and (2) another example to explain behavior when some values is "accidentally" ignored?


The first can just have three values B, A, C. 
The second can have 2-3 more (D,E,F?).  
No need for 9 categories -- only make it more confusing.